symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -678,6 +678,10 @@ void test_constants()
     SYMENGINE_C_ASSERT(strcmp(s, "EulerGamma") == 0);
     basic_str_free(s);
 
+    // Rejecting non mpfr builds
+    s = "mpfr";
+    SYMENGINE_C_ASSERT(symengine_have_component(s));
```

Comment:
As mentioned in the comment, this will fail when `mpfr` is not found. Use `HAVE_SYMENGINE_MPFR` and similar macros to test.